### ISSUE TITLE
Capture webpage with device frame

### ISSUE THREAD

**Issue by nsisodiya** [sindresorhus/pageres#367]
It will be great to have a feature like Adding a device frame while capturing the webpage.
something like this.

![image](https://user-images.githubusercontent.com/333455/57824648-6180d900-77b9-11e9-9a33-9564a1dcbe63.png)

**Comment by sindresorhus** [comment]
Yes, that would be nice indeed. I would only really care about the latest iPhone, iPad, and some of the latest most popular Android phones.

Would need to be implemented in https://github.com/sindresorhus/capture-website, which is the underlying library.

**Comment by nsisodiya** [comment]
@sindresorhus If you provide little help, I can try to add this feature. Maybe like which libraries I should use to join two images (actual image + outside image of the device frame) or similar thing.

**Comment by nsisodiya** [comment]
I have generated a pull request to add this feature. https://github.com/sindresorhus/capture-website/pull/27

**Comment by sindresorhus** [comment]
Going to pass on this.
